Vinyl/Extensions/Bundle.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 // Idea taken from Venmo/DVR (thanks!)
-func testingBundle() -> Bundle {
+public func testingBundle() -> Bundle {
```

Comment:
This needs to be public to be used as a default argument to a public initializer.